### ISSUE TITLE
Jaws 2022 doesn't read Graphics

### ISSUE THREAD

**Issue by ThaMoreira** [FreedomScientific/VFO-standards-support#675]
### Summary
 
Example:
* 1. Go to https://sapui5.hana.ondemand.com/sdk/#/entity/sap.suite.ui.commons.ChartContainer/sample/sap.suite.ui.commons.sample.ChartContainerSimpleToolbar or to https://sapui5.hana.ondemand.com/sdk/#/entity/sap.suite.ui.commons.ChartContainer/sample/sap.suite.ui.commons.sample.ChartContainerToolbarEnhancement
* 2.  Use Jaws to read the Graphic information

### Expected result

Jaws reads correctly the graphic informations

### Actual result

 Jaws only read as " List Box Grapic"

### Example

<img width="885" alt="example" src="https://user-images.githubusercontent.com/36004615/199801606-9c81e641-b5bd-45c2-a3be-b5609b07a9fc.png">

<img width="957" alt="Untitled" src="https://user-images.githubusercontent.com/36004615/199801633-9bb91b90-d875-462f-b015-50af7230aeeb.png">

**Comment by JAWS-test** [comment]
The source code of the SVG file contains an unlabeled selection list (role=listbox) at that position. Therefore, the JAWS output is correct. The rest of the graphic is output correctly by JAWS in conjunction with Chrome. I just had the strange behavior that on the first linear read everything after the selection list is not output, but on the second it is. However, the output of the text contents does not make sense. Either the complete SVG graphic is marked up correctly (with ARIA roles for tables, for example) or the content of the SVG graphic is hidden (aria-hidden=true) and a text alternative in table form is offered underneath. The graphic itself should then only receive a concise alternative text. See https://www.w3.org/WAI/tutorials/images/complex/

**Comment by JAWS-test** [comment]
> But Jaws only reads "List Box Graphic" when it should read the Gross Profit, for example (April-Maried). The same happens with you? Or it read correctly?

Your links do not point to the pages you show in your screenshots. That's why I can't test it. In both of your links, I can open a chart that has a legend with only one entry. But since the legend is marked with role=listbox and role=option, it is normal that its content is not output with the virtual cursor of JAWS. The content of a listbox is output only in form mode, i.e. when I navigate to it with the TAB key. Other screen readers do this differently and output all list entries even when reading linearly. But this has nothing to do with the SVG graphic. A separate ticket should be opened for this if needed. 

The second link opens an SVG graphic with a text alternative in table form. In this example, the graphic should only have a short alternative text and the rest should be marked with aria-hidden=true. The short alternative text should link to the table.

**Comment by ThaMoreira** [comment]
Thank you! I Will open another issue related to the legend reading
